tests/framework/clients/rancher/v1/client.go

```diff
@@ -312,7 +313,13 @@ func (c *NamespacedSteveClient) PerformPutCaptureHeaders(host, token, name strin
 	req.Header.Set("Authorization", "Bearer "+token)
 	req.Header.Set("Content-Type", "application/json")
 
-	var httpClient = &http.Client{}
+	// Create a custom HTTP client with custom transport settings to skip certificate verification
+	tr := &http.Transport{}
+	if skipCertVerification {
```

Comment:
We don't even need to this. We can reference the restConfig from the client, and see if insecure is set. https://github.com/rancher/rancher/blob/c3411e0ca574b7086308bf4ef2e49984cd247dd1/tests/framework/clients/rancher/client.go#L105 make it much simpler

Reply:
Implemented suggestion, thank you! 😁